web3/providers/eth_tester/main.py

```diff
@@ -1,23 +1,45 @@
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Coroutine,
+    Dict,
+)
+
+from web3._utils.compat import (
+    Literal,
+)
 from web3.providers import (
     BaseProvider,
 )
+from web3.types import (
+    RPCEndpoint,
+    RPCResponse,
+)
 
 from .middleware import (
     default_transaction_fields_middleware,
     ethereum_tester_middleware,
 )
 
+if TYPE_CHECKING:
+    from eth_tester import (  # noqa: F401
+        EthereumTester,
+    )
+
 
 class AsyncEthereumTesterProvider(BaseProvider):
     """This is a placeholder.
 
     For now its purpose is to provide an awaitable request function
     for testing the async api execution.
     """
-    def __init__(self):
+    def __init__(self) -> None:
         self.eth_tester = EthereumTesterProvider()
 
-    async def make_request(self, method, params):
+    async def make_request(
+        self, method: RPCEndpoint, params: Any
+    ) -> Coroutine[Any, Any, RPCResponse]:
```

Comment:
I'm confused by this returning a `Coroutine` type.  I would have expected just `RPCResponse`...